### ISSUE TITLE
[0.7.02][WEBSTORM 2023.3] com.intellij.diagnostic.PluginException: 318 ms to call on EDT CreateCommitMessageAction#Up...

### ISSUE THREAD

**Issue by jhonjtoloza** [Codium-ai/codiumai-jetbrains-release#157]
# User Description
_User did not enter any detailed information._

# Stack Trace
```
com.intellij.diagnostic.PluginException: 318 ms to call on EDT CreateCommitMessageAction#Update@ChangesView.CommitToolbar (ai.codium.ui.actions.CreateCommitMessageAction). Revise AnAction.getActionUpdateThread property [Plugin: codiumai.codiumai]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:89)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.computeOnEdt(ActionUpdater.kt:223)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.access$computeOnEdt(ActionUpdater.kt:71)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater$computeOnEdt$1.invokeSuspend(ActionUpdater.kt)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:115)
	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:103)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
	Suppressed: java.lang.Throwable: EDT-trace-at-316-ms (2/5)
		at java.base@17.0.9/java.security.AccessController.getStackAccessControlContext(Native Method)
		at java.base@17.0.9/java.security.AccessController.getContext(AccessController.java:1003)
		at java.desktop/java.awt.Component.<init>(Component.java:512)
		at java.desktop/java.awt.Container.<init>(Container.java:300)
		at java.desktop/javax.swing.JComponent.<init>(JComponent.java:641)
		at java.desktop/javax.swing.JPanel.<init>(JPanel.java:85)
		at java.desktop/javax.swing.JPanel.<init>(JPanel.java:98)
		at com.intellij.ui.BalloonImpl$MyComponent.<init>(BalloonImpl.java:1815)
		at com.intellij.ui.BalloonImpl.createComponent(BalloonImpl.java:746)
		at com.intellij.ui.BalloonImpl.show(BalloonImpl.java:542)
		at com.intellij.ui.BalloonImpl.show(BalloonImpl.java:448)
		at com.intellij.openapi.wm.impl.status.ProcessBalloon.show(ProcessBalloon.java:89)
		at com.intellij.openapi.wm.impl.status.ProcessBalloon.addIndicator(ProcessBalloon.java:45)
		at com.intellij.openapi.wm.impl.status.InfoAndProgressPanel.addProgress(InfoAndProgressPanel.kt:247)
		at com.intellij.openapi.wm.impl.status.IdeStatusBarImpl.addProgress(IdeStatusBarImpl.kt:462)
		at com.intellij.openapi.progress.impl.BackgroundableProcessIndicator.doBackground(BackgroundableProcessIndicator.java:177)
		at com.intellij.openapi.progress.impl.Back

<TRIMMED STACKTRACE>

.EventQueue$3.run(EventQueue.java:739)
		at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:733)
		at java.base@17.0.9/java.security.AccessController.executePrivileged(AccessController.java:776)
		at java.base@17.0.9/java.security.AccessController.doPrivileged(AccessController.java:399)
		at java.base@17.0.9/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
		at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:761)
		at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:695)
		at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$12(IdeEventQueue.kt:589)
		at com.intellij.ide.IdeEventQueue$$Lambda$953/0x000000700092e338.run(Unknown Source)
		at com.intellij.openapi.application.impl.RwLockHolder.runWithoutImplicitRead(RwLockHolder.kt:44)
		at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:589)
		at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:72)
		at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:355)
		at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:354)
		at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:793)
		at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:354)
		at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:349)
		at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:1014)
		at com.intellij.ide.IdeEventQueueKt$$Lambda$948/0x0000007000929df8.run(Unknown Source)
		at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:106)
		at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1014)
		at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:349)
		at com.intellij.ide.IdeEventQueue$$Lambda$947/0x0000007000921a98.run(Unknown Source)
		at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:848)
		at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:391)
		at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
		at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
		at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
		at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
		at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
		at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)

```

# Other Information
```
=== About ===
Build version: WebStorm 2023.3 Build: #WS-233.11799.229 December 1, 2023
Theme: Xcode Dark
JRE: 17.0.9+7-b1087.7, JetBrains s.r.o.
JVM: 17.0.9+7-b1087.7, OpenJDK 64-Bit Server VM, JetBrains s.r.o.
Operating System: Mac OS X 14.1.2 (aarch64)

=== Plugins ===
Custom plugins: [CodeGPT (2.2.9), Atom One Theme (1.5.1), Nest (1.2.22), Dev Containers (233.11799.244), Grazie Pro (0.3.266), Xcode Theme (1.6.3), Toml (233.11799.172), CodiumAI - Integrity Agent powered by GPT-3.5&4 (0.7.02), Vite Integrated (0.2.2), Codeium: AI Autocomplete and Chat for Python, JS, TS, Java, Go… (1.4.22), AI Assistant (233.11799.241), One Dark theme (5.9.0), Astro (233.11799.172), AWS Toolkit - Amazon Q, CodeWhisperer, and more (2.1-233), Godot Theme (1.1.0), Cypress Support (1.6.2), Elegant theme (1.2.2)]
Disabled plugins:[]
```

**Comment by ArtsiomCh** [comment]
Thanks for reporting! We are working on it.

**Comment by ArtsiomCh** [comment]
Will be fixed in 0.7.05
Thanks again for reporting!
